CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 There is no change in how the `styles` option is configured.
 * `i18n` module now uses the regular `req.redirect` instead of a direct `res.redirect` to ensure redirection, enabling more possibilities for `@apostrophecms/redirect` module
 * Refactors `AposModal` component with composition api to get rid of duplicated code in `AposFocusMixin` and `AposFocus`.
+* `APOS_MONGODB_LOG_LEVEL` has been removed. According to [mongodb documentation](https://github.com/mongodb/node-mongodb-native/blob/main/etc/notes/CHANGES_5.0.0.md#mongoclientoptionslogger-and-mongoclientoptionsloglevel-removed) "Both the logger and the logLevel options had no effect and have been removed."
+* Update `mongodb` to `6.x` and `connect-mongo` to `5.x`. Add `@apostrophecms/emulate-mongo-3-driver` dependency to keep supporting `mongodb@3.x` queries.
```

Comment:
* Why do we still need a dependency on `mongodb` at all?
* We're forcing the upgrade, originally we said this would be optional. But I think forcing it will be OK because it looks like you provided good coverage for the 3.x expectations people are likely to have and we are doing it to address compatibility with a supported driver, which has historically been acceptable. Also we did it this way for the 2.x emulator and nobody had a problem with that. So let's go ahead.